lib/bump/update_checkers/ruby/bundler.rb

```diff
@@ -5,6 +5,27 @@
 require "bump/shared_helpers"
 require "bump/errors"
 
+# Allow hijacking Bun
+# rubocop:disable Style/ClassAndModuleChildren
+module ::Bundler
+  class << self
+    alias original_root root
+
+    def hijacked_root(dir)
```

Comment:
You don't need to go to all this bother - you can get away with just doing `::Bundler.instance_variable_set(:@root, Pathname.new(dir))` in the forked process.